From ab0937f548ca893e470fe9d44909f223b485f984 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Mon, 17 Mar 2014 06:29:51 +0100
Subject: [PATCH 366/390] rtl8192du: drop unused function
 update_bcn_htinfo_ie()

This falls through to the default condition and returns.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 core/rtw_ap.c | 8 --------
 1 file changed, 8 deletions(-)

--- a/drivers/staging/rtl8192du/core/rtw_ap.c
+++ b/drivers/staging/rtl8192du/core/rtw_ap.c
@@ -1481,11 +1481,6 @@ static void update_bcn_erpinfo_ie(struct
 	}
 }
 
-static void update_bcn_htinfo_ie(struct rtw_adapter *padapter)
-{
-	DBG_8192D("%s\n", __func__);
-}
-
 static void update_bcn_rsn_ie(struct rtw_adapter *padapter)
 {
 	DBG_8192D("%s\n", __func__);
@@ -1601,9 +1596,6 @@ void update_beacon(struct rtw_adapter *p
 	case _RSN_IE_2_:
 		update_bcn_rsn_ie(padapter);
 		break;
-	case _HT_ADD_INFO_IE_:
-		update_bcn_htinfo_ie(padapter);
-		break;
 	case _VENDOR_SPECIFIC_IE_:
 		update_bcn_vendor_spec_ie(padapter, oui);
 		break;
